packages/mobile/src/components/now-playing-drawer/PlayButton.tsx

```diff
@@ -39,15 +44,38 @@ const useAnimations = makeAnimations(({ palette, type }) => {
     'layers.2.shapes.0.it.1.c.k': palette.primary
   })
 
-  return [ColorizedPlayIcon, ColorizedPauseIcon]
+  const ColorizedSpinnerIcon = IconLoadingSpinner
```

Comment:
TODO: do I need to colorize this too? this json stuff is still unfamiliar to me

Reply:
Yeah it's needed for theming, try on dark mode and you'll see it's the wrong color